server.go

```diff
@@ -1,19 +1,26 @@
 package prunner
 
 import (
-	"encoding/json"
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"time"
 
 	"github.com/apex/log"
 	"github.com/go-chi/chi/v5"
 	"github.com/go-chi/chi/v5/middleware"
 	"github.com/go-chi/jwtauth/v5"
 	"github.com/gofrs/uuid"
+	jsontime "github.com/liamylian/jsontime/v2/v2"
 	"networkteam.com/lab/prunner/taskctl"
 )
 
+var json = jsontime.ConfigWithCustomTimeFormat
+
+func init() {
+	jsontime.SetDefaultTimeFormat(time.RFC3339, time.Local)
```

Comment:
Shouldn't that be UTC for JSON dates?

Reply:
And then assert `require.Equal(t, job.Created.In(time.UTC).Format(time.RFC3339), result2.Jobs[0].Created)`

Reply:
@hlubek good point, resolved :)